pkgs/development/ocaml-modules/dose3/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, buildDunePackage, fetchFromGitLab
+, camlzip, ocamlgraph, parmap, re, stdlib-shims
+, base64, bz2, ocaml_extlib, cudf
+, dpkg, odoc, ounit, python39, python39Packages
+}:
+
+buildDunePackage rec {
+  pname = "dose3";
+  version = "6.1";
+
+  src = fetchFromGitLab {
+    owner = "irill";
+    repo = "dose3";
+    rev = version;
+    sha256 = "sha256-RFChY7VH2pVD4u5U1qng46h9aAv8I/3yXVaNmFDmKFI=";
+  };
+
+  minimalOCamlVersion = "4.03";
+  useDune2 = true;
+
+  buildInputs = [
+    ounit
+    base64
+    cudf
+    parmap
+    re
+  ];
+  propagatedBuildInputs = [
+    bz2
+    camlzip
+    ocaml_extlib
+    ocamlgraph
+    stdlib-shims
+  ];
+
+  nativeCheckInputs = [
```

Comment:
What is this `nativeCheckInputs` attribute? Looks like dead code.

Reply:
Removed